src/functions/getAllCommands.ts

```diff
@@ -0,0 +1,6 @@
+export function getAllCommands(message: any) {
+  message.reply(`My current command list is as follows:
+    - **!ping** <Checks connectivity and replies with pong> :ping_pong:
```

Comment:
Agree that it's not the best looking way of displaying, I saw it on other bots that's why I gave it a try. We could use quotes instead to create some separation between the command and description. We'll go through the output together on Discord and decide.